dart/biomechanics/MarkerFixer.cpp

```diff
@@ -828,7 +829,7 @@ std::shared_ptr<MarkersErrorReport> MarkerFixer::generateDataErrorsReport(
     }
     std::string bestLabel = traces[i].getBestLabel(alreadyTakenLabels);
     traces[i].filterTimestepsBasedOnAcc(dt, 1000.0);
-    traces[i].filterTimestepsBasedOnProlongedStillness(dt, 0.001, 10);
+    // traces[i].filterTimestepsBasedOnProlongedStillness(dt, 0.001, 10);
```

Comment:
Ah, that's right. We added this recently to handle some of the trials with bad marker data in Sam's dataset. I think for now it makes sense to remove this, since the check is probably too aggressive in general. However, it could be nice to re-implement this check in some fashion when we no the trial is not a static trial.

Reply:
Ok, I'll refactor to expose this down to Python, so we can add toggles to AddB's web UI